packages/devtools/lib/src/debugger/debugger.dart

```diff
@@ -1079,16 +1172,97 @@ class ScriptsView implements CoreElementView {
     _items.setRenderer((ScriptRef scriptRef) {
       final String uri = scriptRef.uri;
       final String name = uriDescriber(uri);
-      final CoreElement element = li(text: name, c: 'list-item');
+
+      CoreElement element;
+      if (_matcherRendering != null && _matcherRendering.active) {
+        // InputElement's need to fetch the value not text/textContent property.
+        // The value and text are different, all nodes have a text. It the text
+        // content of the node itself along with its descendants. However, input
+        // elements have a value property - its the input data of the input
+        // element. Input elements may have a text/textContent but it is alway
+        // empty because they are void elements.
+        final html.InputElement inputElement =
+            (_matcherRendering._textfield.element as html.InputElement);
+        final String matchPart = inputElement.value;
+
+        // Bold the characters that match.
```

Comment:
the last characters in the name are the most important so should be bolded.
For example, if the query is "all"
gallery/all.dart
should bold the all in "all.dart" not the "all" in gallery. Switching to lastIndexOf should handle this.

Reply:
Done.